safe/definitions/post_processors.py

```diff
@@ -272,6 +315,81 @@ def post_processor_population_displacement_function(
 
 # A postprocessor can be defined with a formula or with a python function.
 
+post_processor_fatality_ratio = {
+    'key': 'post_processor_fatality_ratio',
+    'name': tr('Population Fatality Ratio Post Processor'),
+    'description': tr(
+        'A post processor to add the population fatality ratio according '
+        'to the hazard class'),
+    'input': {
+        # Taking hazard classification
+        'classification': {
+            'type': keyword_input_type,
+            'value': ['hazard_keywords', 'classification']
+        },
+        'hazard_class': {
+            'type': field_input_type,
+            'value': hazard_class_field,
+        },
+        'population': [
+            # No one of these fields are used in this postprocessor.
+            # But we put them as a condition for the postprocessor to run.
+            {
+                'value': population_count_field,
+                'type': field_input_type,
+            },
+            {
+                'value': exposure_count_field,
+                'field_param': exposure_population['key'],
+                'type': dynamic_field_input_type,
+            }],
+        'earthquake_hazard': {
+            'type': keyword_value_expected,
+            'value': ['hazard_keywords', 'hazard'],
+            'expected_value': hazard_earthquake['key']
+        },
+    },
+    'output': {
+        'fatality_ratio': {
+            'value': population_fatality_ratio_field,
+            'type': function_process,
+            'function': post_processor_population_fatality_function
+        }
+    }
+}
+
+post_processor_fatalities = {
+    'key': 'post_processor_fatalities',
+    'name': tr('Fatalities Post Processor'),
+    'description': tr(
+        'A post processor to calculate the number of fatalities. '),
+    'input': {
+        # input as a list means, try to get the input from the
+        # listed source. Pick the first available
+        'population': [
+            {
+                'value': population_count_field,
+                'type': field_input_type,
+            },
+            {
+                'value': exposure_count_field,
+                'field_param': exposure_population['key'],
+                'type': dynamic_field_input_type,
+            }],
+        'fatality_ratio': {
+            'type': field_input_type,
+            'value': population_fatality_ratio_field
+        },
+    },
+    'output': {
+        'fatalities': {
+            'value': fatalities_field,
+            'type': formula_process,
+            'formula': 'population * fatality_ratio / 100'
```

Comment:
Why is this divided by 100? sIs fatality ratio using percent? not 0.0 - 1.0 like the other ratio?